doc/api/HostDB.md

```diff
@@ -18,10 +18,11 @@ identifies hosts by their public key and keeps track of metrics such as price.
 Index
 -----
 
-| Request                                     | HTTP Verb | Examples                      |
-| ------------------------------------------- | --------- | ----------------------------- |
-| [/hostdb/active](#hostdbactive-get-example) | GET       | [Active hosts](#active-hosts) |
-| [/hostdb/all](#hostdball-get-example)       | GET       | [All hosts](#all-hosts)       |
+| Request                                                 | HTTP Verb | Examples                      |
+| ------------------------------------------------------- | --------- | ----------------------------- |
+| [/hostdb/active](#hostdbactive-get-example)             | GET       | [Active hosts](#active-hosts) |
+| [/hostdb/all](#hostdball-get-example)                   | GET       | [All hosts](#all-hosts)       |
+| [/hostdb/hosts/___:pubkey___](#hostdbhosts-get-example) | GET       | [Hosts](#hosts)               |
```

Comment:
Because the other calls are /hostdb/active, I was considering just making this /hostdb/:pubkey - wasn't sure though.